plugins/hypervisors/vmware/src/com/cloud/hypervisor/vmware/resource/VmwareResource.java

```diff
@@ -1895,6 +1896,9 @@ protected StartAnswer execute(StartCommand cmd) {
 
             postDiskConfigBeforeStart(vmMo, vmSpec, sortedDisks, ideControllerKey, scsiControllerKey, iqnToPath, hyperHost, context);
 
+            //Sets video card memory to the one provided in detail svga.vramSize (if provided), 64MB was always set before
```

Comment:
I think a comment like this would be better located in a Javadoc style comment above the `postVideoCardMemoryConfigBeforeStart` method itself, along with brief descriptions of the parameters involved. This could make it easier to read/understand going forward. :)